source/fem-dem/gls_vans.cc

```diff
@@ -258,6 +258,9 @@ GLSVANSSolver<dim>::calculate_void_fraction(const double time)
       else if (this->cfd_dem_simulation_parameters.void_fraction->mode ==
                Parameters::VoidFractionMode::qcm)
         quadrature_centered_sphere_method();
+      else if (this->cfd_dem_simulation_parameters.void_fraction->mode ==
+               Parameters::VoidFractionMode::spm)
```

Comment:
If you decide to change the name of the parameters, don't forget to do it here also